modules/loader/src/main/java/com/opengamma/strata/loader/csv/EtdFuturePositionCsvPlugin.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2021 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.loader.csv;
+
+import java.util.Optional;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+import com.opengamma.strata.collect.io.CsvRow;
+import com.opengamma.strata.product.Position;
+import com.opengamma.strata.product.PositionInfo;
+import com.opengamma.strata.product.SecurityPosition;
+import com.opengamma.strata.product.etd.EtdFuturePosition;
+
+/**
+ * Handles the CSV file format for CDS index trades.
```

Comment:
Very minor but the JavaDoc for the different position csv plugin classes all refer to CDS index trades